Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -172,14 +172,20 @@ public class Fuzzer {
     ///
     /// This must be called after initializing the fuzzer.
     /// Use -1 for maxIterations to run indefinitely.
-    public func start(runFor maxIterations: Int) {
+    public func start(runFor maxIterations: Int, corpusProgs initalCorpusProgs: [Program]) {
```

Comment:
I think the way we should implement this is by having the user call `importCorpus` manually after initializing but before starting the fuzzer. The log messages can just go into `importCorpus`

Reply:
I think then we also want to move `self.runStartupTests()` to the end of `initialize()`

Reply:
Actually I think that 2nd comment wouldn't work because `initialize()` is used in tests, which don't have access to a real JS engine. I guess we can worry about the proper placement of that call some other time :)

Reply:
I guess that depends on whether main.swift keeps the list alive or not. I think it would happen in this block: https://github.com/googleprojectzero/fuzzilli/blob/master/Sources/FuzzilliCli/main.swift#L223 and so when the block returns, nothing will keep the list alive and the unused programs will be freed. So it should be fine